### DISCUSSION TITLE
Big « blank » space in the Panel

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#34] [General]
Hi,
For the organ I'm working on, I create my stops and panel. 

But when I compose the panel, my stops seems to be too many as I don't have complete list on the columns ... but I have many lost space .. is it possible to "up" the stops buttons ? I tried to do it but it doesn't work 

![image](https://github.com/larspalo/GOODF/assets/20370038/3dc59c8e-21b0-4a25-9ac8-c57c5c4dd56b)

My background is a picture 1280*1024 and in the Displaymetrics when I put the size, pedalboard does not appear

![image](https://github.com/larspalo/GOODF/assets/20370038/8b180ec2-53ef-40b2-8cf8-96ac8c05c204)

It appear at 850 on Vertical panelsize. 
In the Image panel of my background, the maximum is 1280*850 I can't put more ..
I confirm the background is 1280*1024 and it is my screen resolution

**Comment by larspalo** [comment]
Have a look at the number of drawstop rows and columns (mostly rows in your case) and check that the values are suitable for what you want to have on your display and what is actually set for the drawstops. It also seems that your values for the drawstop heights in the displaymetrics are off if you're using the built in row/column layouts.

Remember that you also can utilize the PositionX and PositionY options of manually placing the elements with x and y coordinates on the panel. And if that's the case, just check what the coordinates are for the elements in question.
